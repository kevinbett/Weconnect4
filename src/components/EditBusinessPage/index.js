import React from 'react';
import Navigation from '../Navigation/index';
import EditBusinessBody from '../EditBusinessBody';


const EditBusinessPage = () => (

   <div>
       <Navigation />
       <EditBusinessBody />
      
   </div>
);

export default EditBusinessPage;