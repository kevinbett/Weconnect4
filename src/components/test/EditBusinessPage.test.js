import EditBusinessPage from "../EditBusinessPage"
import { shallow } from "enzyme"

describe('Edit Business page',() => { 
    it ('renders Edit Business page', () => { 
        expect(shallow(<EditBusinessPage/>)).toMatchSnapshot()
    });
} )