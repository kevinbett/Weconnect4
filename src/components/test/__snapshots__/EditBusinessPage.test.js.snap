// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Business page renders Edit Business page 1`] = `
<div>
  <Navigation />
  <EditBusinessForm />
</div>
`;
